diffs/auditbeat2.diff

```diff
@@ -0,0 +1,1923 @@
+@@ -520,7 +520,7 @@
+             <div>
+              <div>
+               <h3 class="title">
+-               <a id="_literal_protocol_literal">
++               <a id="_protocol">
+                </a>
+                <code class="literal">
+                 protocol
+@@ -557,7 +557,7 @@
+             <div>
+              <div>
+               <h3 class="title">
+-               <a id="_literal_proxy_url_literal_4">
++               <a id="_proxy_url_4">
+                </a>
+                <code class="literal">
+                 proxy_url
+@@ -582,7 +582,7 @@
+             <div>
+              <div>
+               <h3 class="title">
+-               <a id="_literal_timeout_literal_5">
++               <a id="_timeout_5">
+                </a>
+                <code class="literal">
+                 timeout
+@@ -607,7 +607,7 @@
+             <div>
+              <div>
+               <h3 class="title">
+-               <a id="_literal_ssl_literal_5">
++               <a id="_ssl_5">
+                </a>
+                <code class="literal">
+                 ssl
+@@ -638,7 +638,7 @@
+             <div>
+              <div>
+               <h3 class="title">
+-               <a id="_literal_username_literal_3">
++               <a id="_username_3">
+                </a>
+                <code class="literal">
+                 username
+html/en/beats/auditbeat/master/configuration-output-codec.html
+html/en/beats/auditbeat/master/configuration-path.html
+--- /dev/fd/63
++++ html/en/beats/auditbeat/master/configuration-path.html
+@@ -211,7 +211,7 @@
+            config file:
+           </p>
+           <h4>
+-           <a id="_literal_home_literal">
++           <a id="_home">
+            </a>
+            <code class="literal">
+             home
+@@ -230,7 +230,7 @@
+            <pre class="programlisting prettyprint lang-yaml">path.home: /usr/share/beats</pre>
+           </div>
+           <h4>
+-           <a id="_literal_config_literal">
++           <a id="_config">
+            </a>
+            <code class="literal">
+             config
+@@ -249,7 +249,7 @@
+            <pre class="programlisting prettyprint lang-yaml">path.config: /usr/share/beats/config</pre>
+           </div>
+           <h4>
+-           <a id="_literal_data_literal">
++           <a id="_data">
+            </a>
+            <code class="literal">
+             data
+@@ -286,7 +286,7 @@
+            </div>
+           </div>
+           <h4>
+-           <a id="_literal_logs_literal">
++           <a id="_logs">
+            </a>
+            <code class="literal">
+             logs
+html/en/beats/auditbeat/master/configuration-ssl.html
+--- /dev/fd/63
++++ html/en/beats/auditbeat/master/configuration-ssl.html
+@@ -177,6 +177,7 @@
+              <a class="link" href="setup-kibana-endpoint.html" title="Configure the Kibana endpoint">
+               Kibana endpoint
+              </a>
++             endif::["auditbeat"!="apm-server"]

```

Comment:
fix required

Reply:
fixed in https://github.com/elastic/beats/commit/de9df2f1a70c47d23093a62b10673a7913765efc